schedule/virtualization/vmware_hyperv.yaml

```diff
@@ -10,9 +10,8 @@ schedule:
     - virtualization/universal/upgrade_guests
     - virtualization/universal/patch_guests
     - '{{open_vm_tools}}'
-
 conditional_schedule:
     open_vm_tools:
         REGRESSION:
             vmware:
-                - virtualization/universal/open_vm_tools
+                - virt_autotest/esxi_open_vm_tools
```

Comment:
@nanzhg  Hi Nan, based on final agreement with PJMs in description of https://progress.opensuse.org/issues/103266, for sle12sp5 and sle15sp4+, new test is used, while for older products sle15sp[1-3] and sle12sp[2-4] , the old test is used. So will need a `if else` here.

Reply:
Hi Alice, I've made the changes, please review. Thanks

Reply:
Thanks @nanzhg . It looks good this way. BTW, sle15 is not included in the final confirmed product matrix. Please refer to the ticket's description for final requirement.